api/v1/swagger.yaml

```diff
@@ -4000,6 +4016,13 @@ paths:
                 description: >
                   Max 60 characters; case insensitive; allows alpha-numeric characters,
                   underscore (_) and dash (-).
+              sortBy:
```

Comment:
Good suggestion. I don't think the core Refocus app would actually use this field to sort subjects in the admin console or even in api response... it's more about making it available as an option for the Lens developer to choose to use or not. Go back to the description in the user story and try to come up with some language to add to swagger based on that.